rust/cargo-psibase/src/main.rs

```diff
@@ -102,6 +108,18 @@ struct TestCommand {
     psitest: PathBuf,
 }
 
+#[derive(Subcommand, Debug)]
+enum ConfigCommand {
```

Comment:
`psibase config hosts set prod http://...` is getting awkwardly long.

Also, the ordering seems a bit awkward if we start adding more fields later.

I would consider treating the key as `hosts.prod` instead of making `hosts` a separate subcommand.

A few similar APIs, for reference:
```
git config <name> [<value> [<value-pattern>]]
git config --unset <name> [<value-pattern>]
```
```
pip config [<file-option>] get command.option
pip config [<file-option>] set command.option value
pip config [<file-option>] unset command.option
```
```
npm config set <key>=<value> [<key>=<value> ...]
npm config get [<key> [<key> ...]]
npm config delete <key> [<key> ...]
```

Reply:
Are you okay with merging as is and then improving in a subsequent pr?

Reply:
That approach is more appropriate for missing functionality. I don't see any reason to rush it, and I'd rather get the API hammered out before it's merged. For the record, I think it would have been better not to bundle so many independent updates into one PR.

Reply:
Addressed